data/human/ships.txt

```diff
@@ -183,6 +183,111 @@ ship "Arrow"
 
 
 
+ship "Auxiliary"
+	plural "Auxiliaries"
+	sprite "ship/auxiliaryh"
+	thumbnail "thumbnail/auxiliaryh"
+	attributes
+		category "Transport"
+		licenses
+			Navy
+			Auxiliary
+		"cost" 13720000
+		"shields" 17100
+		"hull" 5900
+		"required crew" 87
+		"bunks" 260
+		"mass" 820
+		"drag" 13.3
+		"heat dissipation" .45
+		"fuel capacity" 900
+		"cargo space" 250
+		"outfit space" 720
+		"weapon capacity" 160
+		"engine capacity" 200
+		weapon
+			"blast radius" 280
+			"shield damage" 2800
+			"hull damage" 1400
+			"hit force" 4200
+	outfits
+		"Sidewinder Missile Launcher" 2
+		"Sidewinder Missile" 100
+		"Quad Blaster Turret" 2
+		"Heavy Anti-Missile Turret" 2
+		
+		"Fusion Reactor"
+		"LP288a Battery Pack"
+		"D94-YV Shield Generator"
+		"D67-TM Shield Generator"
+		"Water Coolant System"
+		"Large Radar Jammer"
+		"Ramscoop"
+		
+		"X4700 Ion Thruster"
+		"X1700 Ion Thruster"
+		"X5200 Ion Steering"
+		"Scram Drive"
+	engine -41 181
+	engine 41 181
+	gun -12 -180
+	gun 12 -180
+	turret 0 -75 "Heavy Anti-Missile Turret"
+	turret 0 -18
+	turret 0 22.5
+	turret 0 73.5 "Heavy Anti-Missile Turret"
+	fighter 0 -75
+	fighter 0 -18
+	fighter 0 22.5
+	fighter 0 73.5
+	leak "leak" 30 50
+	leak "flame" 30 80
+	leak "big leak" 30 30
+	explode "tiny explosion" 20
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description "The Auxiliary is the Republic Navy's logistical backbone. The size and durability of a capital ship but more cargo and passenger space than any other Navy ship, the Auxiliary is used whenever the Navy needs large amounts of troops or cargo transported with a guarantee that they'll reach the destination."
+	description "	The Auxiliary was built with modularity in mind, allowing the ship to be fitted for more cargo or more bunk space depending on the mission, but this modularity makes the ship rather expensive to build, and therefore a rare sight to see."
```

Comment:
description "	The Auxiliary was built with modularity in mind, allowing the ship to be fitted for more cargo or bunk space depending on the mission. This modularity makes the ship rather expensive to build, however, and therefore a rare sight to see."